### ISSUE TITLE
Not getting warnings for flex start and end

### ISSUE THREAD

**Issue by Enivia** [anandthakker/doiuse#151]
- justify-content
- align-items

chrome 71 only support "flex-start/flex-end" value. But not getting warnings.

**Comment by WebMechanic** [comment]
@Enivia about 40 versions later, this should no longer be an issue :D
you might as well close this non-issue :)

**Comment by clshortfuse** [comment]
There's some newer caniuse links for this but it seems some align-* properties are no longer checked.

Regression from: https://github.com/anandthakker/doiuse/commit/a51556fdba46b2c69a280da013958f0006fab36e#diff-ce96c1428056addae5c99b04be7bf60109c8e048dc3bc8ac610bd65f79aa1bbaL13

Seems the old test was removed as well:

https://github.com/anandthakker/doiuse/commit/a51556fdba46b2c69a280da013958f0006fab36e#diff-f3db8bc9b4c249db92b37c4be54c044dfa31b54911db9e91c4ea770cd1bc177f

It seems like the regression from the commit. 

@RJWadley Can you confirm the regression? I don't think this is affected by the pending partial support PR.

**Comment by RJWadley** [comment]
Yeah, if I recall correctly those properties apply to both CSS flex and CSS grid, and so detecting them is not as simple. Would have to do a more involved check to catch those again.

The `gap` property has a check like that, and it could probably be applied here as well.

**Comment by vaakian** [comment]
any progress?
